webapp/src/components/incoming_calls/call_incoming_condensed.tsx

```diff
@@ -30,8 +29,12 @@ export const CallIncomingCondensed = ({call, onWidget = false, joinButtonBorder
     const teammateNameDisplay = useSelector(getTeammateNameDisplaySetting);
     const caller = useSelector((state: GlobalState) => getUser(state, call.callerID));
 
-    useRingingAndNotification(call, onWidget);
     const [onDismiss, onJoin] = useDismissJoin(call.channelID, call.callID);
+    const onContainerClick = useOnChannelLinkClick(call);
+    useRingingAndNotification(call, onWidget);
+
+    // Do not allow clicks in the webapp expanded view because Safari does not let us switch and focus parent.
+    const onWebappExpanded = !isDesktopApp() && window.location.pathname.includes(`${pluginId}/expanded/`);
```

Comment:
To note, I am not opposed to enable the functionality as designed except on Safari. But fine either way.

Reply:
@matthewbirtch Let me know if you're okay shipping the clicking feature on everything except Safari? (I will turn off clicking if the user is using Safari)

Reply:
@matthewbirtch re-ping (might have gotten lost over the weekend)

Reply:
Let's merge this now as-is since it's been tested. If we feel it's useful to do what we've talked about above please make a ticket @matthewbirtch , thanks.

Reply:
Sorry for the delay folks. Been playing catch-up after being off last week. I guess we'll monitor it and see how it feels. If it's needed we can create a follow-up ticket.